lib/src/qonversion.dart

```diff
@@ -206,28 +206,21 @@ class Qonversion {
   ///
   /// See [Offerings](https://qonversion.io/docs/offerings) for more details.
   /// See [Product Center](https://qonversion.io/docs/product-center) for more details.
-  static Future<QOfferings> offerings() async {
-    final offeringsString =
-        await (_channel.invokeMethod<String>(Constants.mOfferings) as FutureOr<String>);
+    static Future<QOfferings> offerings() async {
```

Comment:
This line for some reason  moved out. It should be aligned on a par with the comment lines. Two lines below are OK. I deleted the comment for them
```suggestion
static Future<QOfferings> offerings() async {
```